mandatory/3-playing-computer.js

```diff
@@ -7,36 +7,47 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
-  2. Remove the line that throws the error.
+    [ console.log(b) throws error as b is undefined]
+  2. Remove the line that throws the error. 
+    [commented out the line causing errors]
   3. What is printed to the console?
+    [2, 6, 6, 4, 7, 9, 8, 6, 9, 13, 10, 8]
   4. How many times is "f1" called?
```

Comment:
Its called twice. There are only 5 iterations of the for loop in total. On each loop the conditional checks if the current index is odd or even. The f1 is called only if the index is odd and as the index is only odd twice (1 and 3) the f1 function is only called twice.